scribble-lib/scribble/browser-support.md

```diff
@@ -0,0 +1,41 @@
+## Scribble Browser Support Policy
+
+This document describes the intended browser support range for HTML
+manuals generated with `#lang scribble/manual`.
+
+This policy defines three levels of support:
+
+- Full support (all features). 
+- Partial support (documentation features)
+- Minimal support (reading documentation text)
+
+Any feature must work correctly in all browsers with full support.
+
+Features that are not needed for reading and understanding the
+documentation, but that are intended for developers of the
+documentation or other specialized audiences, such as links to
```

Comment:
"such as links to implementation" - is this the motivating use case for this pull request? (Just curious)